scripts/bash-test-runner.sh

```diff
@@ -2,9 +2,14 @@
 # Runs all files with _test.sh at the end of the filename
 set -euo pipefail
 
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+# shellcheck source=shell/lib/shell.sh
+source "$DIR/../shell/lib/shell.sh"
+
 # Find all files with _test.sh at the end of the filename
 # and run them
-mapfile -t test_files < <(find . -name "*_test.sh" | sort | grep -v "./node_modules")
+mapfile -t test_files < <(find_files_with_extensions "sh" | grep -E "_test.sh$")
```

Comment:
Small improvement to not duplicate file fetching logic, we use the `find_files_with_extensions` helper from `shell.sh`